launch/tier4_control_launch/launch/control.launch.py

```diff
@@ -429,17 +468,24 @@ def add_launch_arg(name: str, default_value=None, description=None):
 
     # component
     add_launch_arg("use_intra_process", "false", "use ROS 2 component container communication")
-    add_launch_arg("use_multithread", "true", "use multithread")
+    add_launch_arg("use_multithread", "false", "use multithread")
```

Comment:
could you please keep default behavior?

```suggestion
    add_launch_arg("use_multithread", "true", "use multithread")
```
?

Reply:
Revisions were made as suggested.
https://github.com/autowarefoundation/autoware.universe/pull/6805/commits/a28212af7ecef811d9f324357f9a62d855792bea#diff-990cc6e7d5246c36ec9621c61ab42a868240d5a02154f9529258efb3af36e620L377-R378